glider_kinematics_gazebo_plugins/kinematics_control_plugins/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>direct_kinematics_gazebo_plugins</name>
+  <version>0.0.1</version>
+  <description>
+    Direct kinematics control plugin
+  </description>
+
+  <maintainer email="woensug.choi.ks@nps.edu">Woensug Choi</maintainer>
+  <author email="woensug.choi.ks@nps.edu">Woensug Choi</author>
+
+  <license>Apache-2.0</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <depend>gazebo_dev</depend>
```

Comment:
I think you have a few more dependencies that we should try to capture here.  The git submodules along with the packages you need from uuv_simulator, etc.